routes/booking_test.go

```diff
@@ -155,18 +154,25 @@ func (suite *AppTestSuite) TestGetAllBookings() {
 	var payload []*model.Booking
 	_ = json.Unmarshal(rr.Body.Bytes(), &payload)
 	assert.Equal(t, 13, len(payload), "testGetAllBookings: incorrect response size")
-	expectedBookings := []*model.Booking{Booking1, Booking2, Booking3, Booking4, Booking5, Booking6, Booking7}
-	for _, expected := range expectedBookings {
-		found := false
-		for _, b := range payload {
-			if b.Equal(expected) {
-				found = true
-			}
-		}
-		if !found {
-			t.Fatalf("testGetAllBookings: %s not found in offring list", expected.ID)
-		}
-	}
+	assert.Contains(t, payload, Booking1)
+	assert.Contains(t, payload, Booking2)
+	assert.Contains(t, payload, Booking3)
+	assert.Contains(t, payload, Booking4)
+	assert.Contains(t, payload, Booking5)
+	assert.Contains(t, payload, Booking6)
+	assert.Contains(t, payload, Booking7)
+	//expectedBookings := []*model.Booking{Booking1, Booking2, Booking3, Booking4, Booking5, Booking6, Booking7}
+	//for _, expected := range expectedBookings {
+	//	found := false
+	//	for _, b := range payload {
+	//		if b.Equal(expected) {
+	//			found = true
+	//		}
+	//	}
+	//	if !found {
+	//		t.Fatalf("testGetAllBookings: %s not found in offring list", expected.ID)
+	//	}
+	//}
```

Comment:
can remove the commented code

Reply:
Oops I keep forgetting these sorry. I'll also make sure to format this time